RctGcs/config.py

```diff
@@ -9,6 +9,9 @@
 from socket import gaierror, gethostbyname
 from typing import Any, Dict, Tuple
 
+from appdirs import AppDirs
```

Comment:
Imports not sorted

Reply:
This should be here per https://peps.python.org/pep-0008/#imports